src/SourceBuild/content/eng/tools/BinaryToolKit/DetectBinaries.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.FileSystemGlobbing;
+
+namespace BinaryToolKit;
+
+public class DetectBinaries
```

Comment:
This file doesn't hold instance data either so I would also make it static or a singleton.